### PR TITLE
[PM-13776] Generator Icon Button labels

### PR DESCRIPTION
**Pull request by nick-livefront**
## 🎟️ Tracking

[PM-13776](https://bitwarden.atlassian.net/browse/PM-13776)

## 📔 Objective

Update aria labels for generate and copy buttons within the generator components
- Using the `appA11yTitle` across all icon buttons
- Updated all labels to be targeted towards the credential type rather than just "password"

## 📸 Screenshots
|Before|After Extension|After Web|
|-|-|-|
| <video src="https://github.com/user-attachments/assets/01187f87-c015-403e-9ba7-04492fcbe781" /> |<video src="https://github.com/user-attachments/assets/5f6d955c-6344-4164-889f-bf59238516ba" />|<video src="https://github.com/user-attachments/assets/2ac910b1-bd2f-46e9-a88f-e691363a31d9" />|

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-13776]: https://bitwarden.atlassian.net/browse/PM-13776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/browser/src/_locales/en/messages.json
Comment (audreyality) [inline apps/browser/src/_locales/en/messages.json:1]: ⚠️ Please include the new messages in the desktop client as well.

File: apps/browser/src/_locales/en/messages.json [hunk 1]
```diff
@@ -107,6 +107,9 @@
   "copyPassword": {
     "message": "Copy password"
   },
+  "copyPassphrase": {
+    "message": "Copy passphrase"
+  },
```
Comment (audreyality) [inline apps/browser/src/_locales/en/messages.json:112]: 🤔 These should probably just be labelled "copy" and "generate". If you want to include the credential type, it should be read from the algorithm info and injected as a parameter into the translation.

File: libs/tools/generator/components/src/credential-generator.component.ts [hunk 2]
```diff
@@ -63,6 +63,36 @@ export class CredentialGeneratorComponent implements OnInit, OnDestroy {
     nav: null,
   });
 
+  /** Emits the copy button aria-label respective of the selected credential type */
+  protected credentialTypeCopyLabel$ = this.root$.pipe(
+    map(({ nav }) => {
+      if (nav === "password") {
```
Comment (audreyality) [inline libs/tools/generator/components/src/credential-generator.component.ts:69]: 🌱 The forwarder PR (#11563) moves translation into the service. Please put `FIXME` comments on each of these labels to move them into the algorithm metadata once that branch lands.

```suggestion
  protected credentialTypeCopyLabel$ = this.root$.pipe(
    // FIXME: add labels to `AlgorithmInfo` and perform translation in the `CredentialGeneratorService`
    map(({ nav }) => {
      if (nav === "password") {
```

The above suggestion is an example; the FIXME should be applied in every location where this code exists, and also as an HTML comment (`<!-- -->`) in the username generator UI where there's a fixed label.

**Review by audreyality — CHANGES_REQUESTED** [review]

File: apps/browser/src/_locales/en/messages.json
Comment (nick-livefront) [reply apps/browser/src/_locales/en/messages.json:1]: 🙇 Thank you! Added [badff32](https://github.com/bitwarden/clients/pull/11623/commits/badff32c0417e7a579b1408ea44ea0c25b51d846)

**Pull request merged**